src/shared/animations/index.js

```diff
@@ -0,0 +1,88 @@
+import welcome from './welcome.json';
+import language from './language.json';
+import ledger from './ledger.json';
+import onboardingComplete from './onboardingComplete.json';
+import get from 'lodash.get';
+
+import themes from '../themes/themes';
+
+export const animations = {
+    language,
+    ledger,
+    onboardingComplete,
+    welcome,
+};
+
+/**
+ * Returns colorised Lottie animation data
+ * @param {string} animationName - Target animation name
+ * @param {object} themeName - Target colors
+ * @returns {object} -Target animation data
+ */
+export const getAnimation = (animationName, themeName) => {
+    const data = animations[animationName];
+    const theme = themes[themeName];
+
+    if (!data) {
+        return null;
+    }
+
+    return theme ? replaceColors(data, theme) : data;
+};
+
+/**
+ * Converts rgb color to vector array
+ * @param {string} color - target color in rgb(255,255,255) form
+ * @returns {array} - color as vector array
+ */
+const rgbToVector = (color) => {
+    if (!color) {
+        return [1, 1, 1, 1];
+    }
+    const precision = 1000000;
+
+    return color
+        .substr(color.indexOf('(') + 1)
+        .split(',')
+        .map((str) => Math.floor(parseInt(str) / 255 * precision) / precision)
+        .concat([1]);
+};
+
+const defaultTones = [
+    'rgb(3,41,62)',
+    'rgb(147,168,172)',
+    'rgb(65,220,243)',
+    'rgb(15,62,87)',
+    'rgb(208,220,238)',
+    'rgb(4,52,81)',
+    'rgb(61,91,107)',
+    'rgb(192,205,225)',
+    'rgb(7,30,39)',
+].map((tone) => rgbToVector(tone));
```

Comment:
Would it be easier to just store the vectors as constants here?

Reply:
Easier, yes, but not human readable :/ Maybe we can move them to documentation.

Reply:
Ah, good point. The RGB values could also be commented inline next to the vector values